chainio/clients/txsender/txsender.go

```diff
@@ -0,0 +1,15 @@
+package txsender
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type TxID = string
+
+// TxSender is an interface for signing and sending transactions to the network
```

Comment:
To be clear, this interface can be used to build a sender that sends tx to a private mempool, but that's not the primary use case. At least for hot wallet implementation, this interface is necessary because transactions are broadcasted via external API, not eth client. I'll add a comment documenting this